src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private String login;
```

Comment:
Also u can use @Column(unique = true) for 
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/135730971/606cbca4-0013-4179-a2d6-ce2b536f087c)